src/command/history.rs

```diff
@@ -2,60 +2,55 @@ use std::env;
 use std::io::Write;
 use std::time::Duration;
 
+use clap::Subcommand;
 use eyre::Result;
-use structopt::StructOpt;
 use tabwriter::TabWriter;
 
 use atuin_client::database::Database;
 use atuin_client::history::History;
 use atuin_client::settings::Settings;
 use atuin_client::sync;
 
-#[derive(StructOpt)]
+#[derive(Subcommand)]
+#[clap(aliases = &["h", "hi", "his", "hist", "histo", "histor"])]
```

Comment:
Instead of all these aliases, `InferSubcommands` setting will provide the same result

Reply:
Thanks! I'll try it in the morning. Thanks for working on getting clap3 out there!